MobileCenterDistribute/MobileCenterDistributeTests/MSBasicMachOParserTests.m

```diff
@@ -0,0 +1,54 @@
+#import <XCTest/XCTest.h>
+#import "MSBasicMachOParser.h"
+#import "MSUtil.h"
+
+@interface MSBasicMachOParserTests : XCTestCase
```

Comment:
I don't see this file in the project while checkout this branch, maybe the project file was not saved properly?